requirements.txt

```diff
@@ -1,16 +1,26 @@
 alembic==0.8.4
+blinker==1.4
```

Comment:
You don't need all dependencies in requirements, they will be installed automatically.
Just add the library you explicitly installed, like:
```
pip install Flask-Mail
pip freeze | grep Flask-Mail >> requirement.txt
Please, try to minimize the requirements file